float64 joint1
float64 joint3
float64 joint5
---
int64 state
